index.py

```diff
@@ -144,3 +188,29 @@ def get_apps():
     
     return render_template('index.html', options=options, organized_list=organized_list )
 
+
+@app.route('/apps/<app_module>/<app_name>/<tag>', methods=['GET'])
+@app.route('/apps/<app_module>/<app_name>', methods=['GET'])
+def get_app(app_module, app_name, tag="release"):
+    app_id = app_module + '/' + app_name
+
+    app_page_payload = {
+        'id': 0,
+        'method': 'NarrativeMethodStore.get_method_full_info',
+        'version': '1.1',
+        'params': [{   
+                    'ids': [app_id],
+                    'tag': tag
+                }]
+    }
+    response = requests.post(_NarrativeMethodStore_url, data=json.dumps(app_page_payload))
+    try:
+        response_json = response.json()
+        # App info is stored in the first element of the result array.
+        app_info = response_json['result'][0][0]
+    except ValueError as err:
```

Comment:
Same as the error above - if you just print the error and continue, then `app_info` won't exist, and the next line will fail as well. Probably best to either transform this into another error that goes to the calling function, or return something else that the calling function can use.

Reply:
Or just render an error.